en/micro-integrator/docs/setup/deployment/migrating-from-ei-7.0.0.md

```diff
@@ -0,0 +1,51 @@
+# Migrating from WSO2 EI 7.0.0 to WSO2 EI 7.1.0
+
+### Set up the migration
+
+-	Make a backup of the existing database used by the current EI 7.0.0 deployment. This backup is necessary in case the migration causes any issues in the existing database.
+-	Download and install EI 7.1.0 in your environment. The home directory of your installation will be referred to as `<MI_HOME>` from hereon.
+	-	[On a VM](../../../setup/installation/install_in_vm)
+	-	[On Docker](../../../setup/installation/run_in_docker)
+	-	[On Kubernetes](../../../setup/installation/run_in_kubernetes)
+-	You can [use WSO2 Update Manager](https://docs.wso2.com/display/updates/) to get the latest available updates.
+
+	!!! Info
+		Note that you need a valid [WSO2 subscription](https://wso2.com/subscription) to use updates in a production environment.
+
+### Migrating databases
+You can use the same databases that you used for the EI 7.0.0 of WSO2 with WSO2 EI 7.1.0. Simply copy the database configurations to the `MI-HOME/conf/deployment.toml` file as you did in WSO2 EI 7.0.0. 
+
+### Migrating the registry
```

Comment:
Shouldn't we be copying the MI_HOME/registry/ folder?

Reply:
I think better we not copy and encourage ppl to deploy capps and not to mess with registry manually

Reply:
we have to mention that in both migration docs. @sdkottegoda  any other opinion ?

Reply:
I feel like the statement 'You can directly migrate the artifacts to the EI 7.1.0 by adding the carbon applications to the `MI-HOME/repository/deployment/server/carbonapps` folder. ' is not very clear here. We should probably explain the Carbon applications as it relates to the registry a bit more. wdyt?

Reply:
https://github.com/wso2/docs-ei/blob/master/en/micro-integrator/docs/setup/deployment/migrating-from-ei-6.x.x.md#migrating-the-registry how about a variant of msg from this